Mathlib/CategoryTheory/Sites/RegularExtensive.lean

```diff
@@ -67,39 +68,6 @@ class Preregular : Prop where
   exists_fac : ∀ {X Y Z : C} (f : X ⟶ Y) (g : Z ⟶ Y) [EffectiveEpi g],
     (∃ (W : C) (h : W ⟶ X) (_ : EffectiveEpi h) (i : W ⟶ Z), i ≫ g = h ≫ f)
 
-/--
-Describes the property of having pullbacks of morphsims into a finite coproduct, where one
-of the morphisms is an inclusion map into the coproduct (up to isomorphism).
--/
-class HasPullbacksOfInclusions : Prop where
-    /-- For any morphism `f : X ⟶ Z`, where `Z` is the coproduct of `i : (a : α) → Y a ⟶ Z` with
-    `α` finite, the pullback of `f` and `i a` exists for every `a : α`. -/
-    has_pullback : ∀ {X Z : C} {α : Type w} (f : X ⟶ Z) {Y : (a : α) → C}
-    (i : (a : α) → Y a ⟶ Z) [Fintype α] [HasCoproduct Y] [IsIso (Sigma.desc i)] (a : α),
-    HasPullback f (i a)
-
-instance [HasPullbacksOfInclusions C] {X Z : C} {α : Type w} (f : X ⟶ Z) {Y : (a : α) → C}
-    (i : (a : α) → Y a ⟶ Z) [Fintype α] [HasCoproduct Y] [IsIso (Sigma.desc i)] (a : α) :
-    HasPullback f (i a) := HasPullbacksOfInclusions.has_pullback f i a
-
-/--
-If `C` has pullbacks then it has the pullbacks relevant to `HasPullbacksOfInclusions`.
--/
-instance (priority := 10) [HasPullbacks C] :
-  HasPullbacksOfInclusions C := ⟨fun _ _ _ => inferInstance⟩
-
-/--
-A category is *extensive* if it has all finite coproducts and those coproducts are preserved
-by pullbacks (we only require the relevant pullbacks to exist, via `HasPullbacksOfInclusions`).
-
-TODO: relate this to the class `FinitaryExtensive`
--/
-class Extensive extends HasFiniteCoproducts C, HasPullbacksOfInclusions C : Prop where
-  /-- Pulling back an isomorphism from a coproduct yields an isomorphism. -/
```

Comment:
I would like this class to remain, under the name `Preextensive`/`PreFinitaryExtensive`. Then we can leave this file as it is and just add an `instance [FinitaryExtensive C] : Preextensive C`